Sources/NIOHTTP2/ConnectionStateMachine/HasExtendedConnectSettings.swift

```diff
@@ -0,0 +1,21 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2024 Apple Inc. and the SwiftNIO project authors
```

Comment:
```suggestion
// Copyright (c) 2024 Apple Inc. and the SwiftNIO project authors
```

Reply:
Also, you'll need to update this line:

https://github.com/apple/swift-nio-http2/blob/8d8eb609929aee75336a0a3d2417280786265868/scripts/soundness.sh#L21

It needs to become:

```
sed -e 's/20[12][7890123]-20[12][8901234]/YEARS/' -e 's/20[12][8901234]/YEARS/' 
```